app/src/main/java/ie/macinnes/tvheadend/player/Player.java

```diff
@@ -119,9 +128,12 @@
     private View mOverlayView;
     private DebugTextViewHelper mDebugViewHelper;
     private SubtitleView mSubtitleView;
+    private LinearLayout mRadioInfoView;
 
     private MediaSource mMediaSource;
 
+    private Uri currentChannelUri;
```

Comment:
The naming pattern we use would make this `mCurrentChannelUri` (I think it pattern came from android's style guide, I can't remember anymore.. heh :))